maquetado/scss/styles.scss

```
@@ -3,7 +3,10 @@
 }
 
 .book-container {
-  padding: 10px;
+  padding: 20px;
+  background-color: #FFF;
+  box-shadow: 4px 4px 10px 0 $shadow-background;
+  width: 1000px;
```

Remember to read our CSS style guide. There it says that elements with a width higher than 250px have to use following structure for responsive purposes: 

```suggestion
max-width: 1000px;
width: 100%;
```